### ISSUE TITLE
Update workspace jupyterlab extension installation commands

### ISSUE THREAD

**Issue by marjo-luc** [MAAP-Project/ZenHub#817]
We currently use something like this to install our custom extensions:
`jupyter labextension install @maap-jupyterlab/algorithms_jupyter_extension@0.1.1 --no-build`

This method is deprecated and will not be supported in the next major version of jupyterlab. We should instead be installing the extensions directly from a package manager (i.e. npm/pip install ...). Our extensions are already published to these registries, so the crux of the work for this ticket will be confirming the updated commands work as expected and determining whether or not we need to update our extension release workflow.

**Comment by anilnatha** [comment]
Related to this, we need to investigate the usage of [npm-shrinkwrap](https://docs.npmjs.com/cli/v10/commands/npm-shrinkwrap) so that our package dependencies are published, and subsequently when we need to `npm install` one of our packages, the "shrinkwrapped" dependency list is used.

This [link](https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json#package-lockjson-vs-npm-shrinkwrapjson) speaks about the differences between `package-lock.json` and `npm-shrinkwrap.json`, while [this link](https://johnguerra.co/lectures/visualAnalytics_12hours/node_modules/npm/html/doc/cli/npm-shrinkwrap.html) goes into detail about how npm-shrinkwrap would be used during our typical development workflow of an npm package.

**Comment by marjo-luc** [comment]
Replacing `jupyter labextension install `with `npm install` does not currently work. [This ticket](https://github.com/jupyterlab/jupyterlab/issues/12386) indicates source extensions will no longer be managed using jupyter labextension in jupyterlab v5, but provides no clear alternative. 

I've tried installing official jupyterlab extensions from source and get the same results as I do with our custom extensions -- everything appears to install as expected with no errors, but the extensions are not registered with the extension manager. I tried installing one of our custom extensions from a prebuilt package and that yielded the same results. I'm reaching out to the jupyter community for input.
